mandatory/4-tax.js

```diff
@@ -17,7 +20,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  price = price + price / 5;
+  return `£${price.toFixed(2)}`;
+}
```

Comment:
Another one on pure functions

Reply:
Just found out what it is
   const newPrice = price + price/5

`...a pure function should not modify its input parameters or any other external variables or state`

Reply:
yes! that would be a better approach. We should try avoiding modification of parameters because if you call the same function multiple times you will keep getting different answers.